From dc47d05328b53f0515dea60f7ec40aa4bb1b4cb0 Mon Sep 17 00:00:00 2001
From: "Thakur, Santosh Krx" <santosh.krx.thakur@intel.com>
Date: Thu, 4 Feb 2016 17:53:14 +0530
Subject: [PATCH 18/24] SoC: Intel: klockwork error in Audio Driver

This patch initializes the "ret" variable.

Change-Id: Ie1ac78346654a67eb51c53dca215cac095ad77a1
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-18673
Signed-off-by: Thakur, Santosh Krx <santosh.krx.thakur@intel.com>
Reviewed-on: https://android.intel.com:443/466507
---
 drivers/input/misc/soc_button_array.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/input/misc/soc_button_array.c b/drivers/input/misc/soc_button_array.c
index 93861b6..1a0189d 100644
--- a/drivers/input/misc/soc_button_array.c
+++ b/drivers/input/misc/soc_button_array.c
@@ -125,7 +125,7 @@ err0:
 static int soc_button_pnp_probe(struct pnp_dev *pdev,
 	const struct pnp_device_id *id)
 {
-	int i, j, r, ret;
+	int i, j, r, ret = 0;
 	int sz_tbl = sizeof(soc_button_tbl) / sizeof(soc_button_tbl[0]);
 	struct gpio_keys_button *gk;
 
-- 
1.9.1

